### ISSUE TITLE
Attached transports do not respect individual minLevel

### ISSUE THREAD

**Issue by rpss1** [fullstack-build/tslog#42]
**Describe the bug**
When transport's `minLevel` is different from stdoutput, the log is not generated on transports with `minLevel` less than stdoutput.

**To Reproduce**
1. Run the following code:
```
import { ILogObject, Logger } from 'tslog';
import { appendFileSync }  from 'fs';

function logToTransport(logObject: ILogObject) {
  appendFileSync('logs.txt', JSON.stringify(logObject) + '\n');
}

const logger: Logger = new Logger({ minLevel: 'info' });
logger.attachTransport(
  {
    silly: logToTransport,
    debug: logToTransport,
    trace: logToTransport,
    info: logToTransport,
    warn: logToTransport,
    error: logToTransport,
    fatal: logToTransport,
  },
  'debug'
);

logger.info("I am an info log.");
logger.warn("I am a warn log with a json object:", { foo: "bar" });
logger.debug("I am a debug log.");
```
2. The "info" log is generated on the stdoutput and attached transport.
3. The "warn" log is generated on the stdoutput and attached transport.
4. The "debug" log is not generated on the attached transport.

**Expected behavior**
The "debug" log should be generated in the attached transport with `minLevel` is `debug`.

**Additional context**
To resolve, I believe that is need to change the code highlighted in the image below from `this.settings.minLevel` to `transport.minLevel`.

File: tslog/src/index.ts

![tempsnip](https://user-images.githubusercontent.com/69718707/90312749-60b78500-dedd-11ea-8575-48599d1e8674.png)

**Comment by terehov** [comment]
Thank you very much. I’ll release an update shortly.

**Comment by terehov** [comment]
I have fixed this in '2.7.1' and added a test for this use case. Thank you once again :-)
